README.md

```diff
@@ -5,40 +5,25 @@
 
 # SMTP Integrator Operator
 
-A [Juju](https://juju.is/) [charm](https://juju.is/docs/olm/charmed-operators)
-deploying and managing a SMTP Integrator on Kubernetes and bare metal. SMTP
-is an Internet standard communication protocol for email transmission.
-
-This charm simplifies configuration of SMTP client by providing a single point
-of configuration for all the requirers using the same SMTP server. It can be
-deployed on many different Kubernetes platforms, from [MicroK8s](https://microk8s.io)
-to [Charmed Kubernetes](https://ubuntu.com/kubernetes), public cloud Kubernetes
-offerings and virtual machines or bare metal.
-
-As such, the charm makes it easy to manage and propagate SMTP configuration, while
-giving the freedom to deploy on the substrate of their choice.
+A [Juju](https://juju.is/) [charm](https://juju.is/docs/olm/charmed-operators) 
+for deploying and managing an SMTP integrator on Kubernetes or bare metal. 
+The SMTP integrator provides SMTP connection information to other charms 
+that require `smtp` integration, facilitating the email-sending capabilities 
+of those charms.
 
 For DevOps or SRE teams this charm will make operating any charm requiring SMTP
 authentication simple and straightforward through Juju's clean interface.
 
```

Comment:
I would add one more sentence to the intro that says `For information about how to deploy, integrate, and manage this charm, see the Official [SMTP Integrator Operator Documentation](https://charmhub.io/smtp-integrator).`